### ISSUE TITLE
The Vivaldi icon needs an update.

### ISSUE THREAD

**Issue by cdysthe** [OrancheloTeam/oranchelo-icon-theme#40]
It would be great if the Vivaldi icon could be updated to the new icon they are using. There should also be a green Vivaldi icon for the Vivaldi snapshots. Many use snapshots  instead of the stable version. The new Vivaldi icon here:

https://is.gd/qSmReE

Thank you for a phenomenal icon pack!

**Comment by zayronxio** [comment]
a new vivaldi icon has been added, however this is different from the one you ask, I try to make the icons different, observe it and let me know if you like it

**Comment by cdysthe** [comment]
Thank you! I really like it! One little issue though. I messed up. Turns out that public snapshot builds should have a black icon. They live [here]( https://vivaldi.com/blog/snapshots/)

The green is reserved for internal master builds. I know that since I run them :) 

So Vivaldi has the stable general release with red icon and Vivaldi-snapshot with black icon. No beta builds at all. Again, love the icon!
